templates/SilverStripe/CMS/Controllers/Includes/CMSMain_TreeView.ss

```diff
@@ -1,9 +1,5 @@
 <% include SilverStripe\\CMS\\Controllers\\CMSMain_ContentToolActions View='Tree' %>
 
-<div class="ss-dialog cms-page-add-form-dialog cms-dialog-content" id="cms-page-add-form" title="<%t SilverStripe\Admin\LeftAndMain.AddNew 'Add new {name}' name=$getRecord('singleton').i18n_singular_name().lowercase %>">
```

Comment:
This used add a hidden and completely redundant CMSMainAddForm inbetween the row with the 'Add new page' button on it and the site tree - however because it contained a TreeDropdownField, which by default does an AJAX request to /tree, it was spamming off pointless and expensive requests to fetch the site tree for no reason.

The actual place this CMSMainAddForm is used is on /admin/pages/add. I suspect people just forgot to delete the old one from /admin/pages.